pyrep/objects/vision_sensor.py

```diff
@@ -193,6 +193,30 @@ def set_render_mode(self, render_mode: RenderMode) -> None:
             render_mode.value
         )
 
+    def get_windowed_size(self) -> Sequence[int]:
+        """Get the size of windowed rendering.
+
+        :return: The (x, y) resolution of the window. 0 for full-screen.
+        """
+        size_x = sim.simGetObjectInt32Parameter(
+            self._handle, sim.sim_visionintparam_windowed_size_x)
+        size_y = sim.simGetObjectInt32Parameter(
+            self._handle, sim.sim_visionintparam_windowed_size_y)
+        return size_x, size_y
+
+    def set_windowed_size(self, resolution: Sequence[int] = (0, 0)) -> None:
+        """Set the size of windowed rendering.
+
+        :param resolution: The (x, y) resolution of the window.
+            0 for full-screen.
+        """
+        sim.simSetObjectInt32Parameter(
+            self._handle, sim.sim_visionintparam_windowed_size_x,
+            resolution[0])
+        sim.simSetObjectInt32Parameter(
+            self._handle, sim.sim_visionintparam_windowed_size_y,
+            resolution[1])
```

Comment:
What if we just move these to the `set_resolution` function? Then the user can use the same function for both windowed and non windowed. Also, would you be able to call this in the `create` function?

Reply:
Maybe making the default window size the same as the resolution would be better, but I'm not sure about merging these into a single function (ie `set_resolution`). Though currently, we can't call `capture_rgb()` with `RenderMode.OPENGL3_WINDOWED`,  if possible, I would set a high resolution for `capture_rgb()` (e.g., 1280x960), and lower resolution for the window (e.g., 640x480), since a big window is bothering.